From 93c124215487fe7fe4ae9cfc518e16fb25b2d5d2 Mon Sep 17 00:00:00 2001
From: Long Trinh <long.trinh.rh@bp.renesas.com>
Date: Mon, 31 Mar 2025 07:03:49 +0000
Subject: [PATCH 1/2] rzv2l-evk : add wm8978 codec to support audio in v2l evk

This commit add a config to support audio in RZ V2L EVK

Upstream-Status: Pending

Signed-off-by: Long Trinh <long.trinh.rh@bp.renesas.com>
---
 arch/arm64/configs/rzv2l_config | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 arch/arm64/configs/rzv2l_config

diff --git a/arch/arm64/configs/rzv2l_config b/arch/arm64/configs/rzv2l_config
new file mode 100644
index 000000000000..a51f4c9439fb
--- /dev/null
+++ b/arch/arm64/configs/rzv2l_config
@@ -0,0 +1 @@
+CONFIG_SND_SOC_WM8978=y
-- 
2.25.1

